src/WorkflowsRules/Contracts/Interfaces/IAction.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Kanvas\Packages\WorkflowsRules\Contracts\Interfaces;
+
+interface IAction
```

Comment:
@FredPeal we dont follow this naming convention we add interface to our interface files